### DISCUSSION TITLE
Attendance Options

### DISCUSSION THREAD

**Discussion by KingTut1762** [code4recovery/12-step-meeting-list#1171] [Q&A]
Greetings,
Thank you, Code4Recovery for all you do, especially for the newcomer!
I'd like to get some clarity regarding the three attendance options. 'In-person', 'Online', and 'In-person and Online'.
I recently contacted a neighboring district's website devs/editors to sync up our meeting schedules posted online. To me the options are self-explanatory and simple. I've seen some varied interpretations of the "In-person and Online" option, so, would anyone out there like to set the record straight on this?
Thank you---

**Comment by FresnoAAWebChair** [comment]
"In-person and Online"  could mean two things - That particular group or meeting can have a "In person"Only meeting and a "Online" Only meeting that is in two different locations but at the same time and day or it could mean it is a "Hybrid" meeting which means they are at the same location, so Hybrid means online meetings will be able to be in the same meeting with the In-Person Meeting
I hope this helps

**Reply by KingTut1762** [reply]
Thank you for responding, and this is exactly what I'm trying to clarify and possibly standardize. This is probably written in text somewhere, but what were the intentions of the TSML devs regarding these options?
During Covid restrictions, our district (and many others) listed meetings that had an address, which meant it was an _**"in-person"**_ meeting. _**"Online"**_ meant through an electronic device with no physical location. _**"In-person and Online"**_ meant a bridging of the two types and was (and still is) referred to as _**"hybrid"**_, a term not widely accepted, but it distinguished it from the other two options.
While your interpretation of an _**"In person" Only meeting and an "Online" Only meeting that is in two different locations but at the same time and day"**_ could mean that the _**"two locations"**_ are physical and non-physical. 
Let's look at a third meaning of the _**"In-person and Online"**_ attendance option.

The following is an **actual** interpretation of the _**"In-person and Online"**_ option:

Our neighboring district has 10 times more meetings listed than ours and uses the "In-person and Online" option in the following way:_

For concurrent meetings, the neighbor district only uses one meeting listing with two options as long as the meeting is the same duration, and the same type (e.g., one hour, open, speaker/discussion), and because they are exactly the same type of meeting they only use one meeting listing and people can choose either to click on Zoom or show up at the location to attend the in-person meeting that is happening concurrently. 

The only time they make a separate meeting listing for concurrent online and in-person meetings is when the meeting format or duration is different. 

For example, if a Friday 5:30pm in-person meeting was a big book study, and the online meeting was a grapevine study, then they would list two separate meetings. 

If a meeting is in fact a hybrid meeting (The video camera set up inside the live meeting) then they actually specify that on the meeting listing that it’s “hybrid”. 

_**Shouldn't this be less confusing? Shouldn't we be making this simple for the newcomer?**_

For sake of argument, the English language is full of ambiguity. The words "their", "there", and "they're" sound the same, but they all mean different things. For someone learning English, obviously this is confusing at first.

In my experience, a confused newcomer goes back to what they think works for them. I can say this because it happened to me. I stayed sick a few more years and I was very fortunate to have given recovery another shot.

I may be picking the "fly poop out of the pepper", but that's exactly all a newcomer needs to find, that one difference that keeps them from looking at a solution to all of their problems. We need them to see the similarities.

**Reply by joshreisner** [reply]
hi, i'm a TSML dev and we've written about this here: 

> Why no "Hybrid" type?
>
> This app exists to help people find AA meetings, and after much discussion we decided that Hybrid was not a useful filter type for that purpose. We believe that people on the whole do not set out looking for a hybrid meeting, they simply want to know whether their online or in-person meeting happens to be hybrid. They will know this by its appearance in the list.
>
> Second, while we can infer that a meeting is "online" if there is a Zoom URL (for example) in the listing, the app should not assume that, when there are online and in-person options, that means it is an actual "hybrid" meeting with a video screen and speakers in the room.
>
> Web servants may [add their own meeting types](https://github.com/code4recovery/tsml-ui#add-custom-types) of course.

as an individual looking for a meeting, i think you and i are aligned. i would expect that meetings listed with both in person and online options like this one have an a/v system in the room:

![both](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/46b6028b-52fb-4b15-88a8-4710c267ec8d)

many groups in san jose choose to conduct parallel meetings and list them separately, this sounds like it's your preference too, and, i think, the norm.

![parallel](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/dc32a873-c14c-4651-8739-ab5657330745)

not sure how we could engineer a solution for your neighboring district. if we started displaying their concurrent / parallel meetings as "hybrid" i'm sure we'd get a stern message from them demanding to know why we suddenly changed it ;) 

also while "hybrid" is the prevailing term for this type of meeting among members, i'm not sure that it actually does make much sense to someone looking at a meeting list for the first time. not like "Online & In-Person" would.

at any rate, i'll flag this for the other developers, perhaps they have more to say on the matter!

**Reply by KingTut1762** [reply]
Agreed, I don't like the term "hybrid" either, it pertains to animal life primarily. The "In-person and Online" is the most accurate to describe a single meeting with a device in the room to bridge the "In-Person" and the "Online".
Many thanks to you Josh, for your dedication and hard work!
